packages/gatsby-page-utils/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Gatsby library that helps creating pages",
   "main": "dist/index.js",
   "scripts": {
-    "build": "babel src --out-dir dist/ --ignore \"**/__tests__\"",
-    "watch": "babel -w src --out-dir dist/ --ignore \"**/__tests__\"",
+    "build": "babel src --out-dir dist/ --ignore **/__tests__ --extensions \".ts,.js\"",
+    "watch": "babel -w src --out-dir dist/ --ignore **/__tests__ --extensions \".ts,.js\"",
```

Comment:
Shouldn't the `**/__tests__` parts be double quoted as well? (They were before)

I think this makes the glob expansion safer in certain cases.

Reply:
Fair. These shouldn't have been lost

Reply:
I suspect you fixed this and then the fix got lost after the @types update? :)